spec/checkPullRequestReviewSpec.js

```diff
@@ -1522,5 +1522,26 @@ describe('Pull Request Review Module', () => {
         });
       }
     );
+
+    describe(
+      'when author asks some question/comment and asks reviewer to ptal',
+      () => {
+        const initialCommentBody = commentPayloadData.payload.comment.body;
+        beforeAll(() => {
+          commentPayloadData.payload.comment.body =
+          '{{Question/comment}} @{{reviewer_username}} PTAL';
+        });
+        beforeEach(async () => {
+          await robot.receive(commentPayloadData);
+        });
+
+        it('should not comment on pull request', () => {
+          expect(github.issues.createComment).not.toHaveBeenCalled();
```

Comment:
I'm finding it difficult to understand what's been tested here! Can you please help me with the link to the test which validates oppiaBot comments when someone leaves a PTAL comment?

Reply:
https://github.com/oppia/oppia/pull/12132#issuecomment-803470984 @DubeySandeep does this one satisfy the test?